packages/web-sdk/src/transports/otlp/transform/Resource.ts

```diff
@@ -0,0 +1,58 @@
+import { TelemetrySdkLanguageValues } from '@opentelemetry/semantic-conventions';
+
+import type { APIEvent, TransportItem } from 'packages/core/dist/types';
+import { attributeValueType, toAttribute, toNestedAttributes } from './attributeUtils';
+import { faroResourceAttributes } from './semanticResourceAttributes';
+import type { Attribute, FaroResourceAttributes, PayloadResourceChild } from './types';
+
+type PayloadResource = {
+  resource: {
+    attributes: Attribute<FaroResourceAttributes>[];
+    droppedAttributesCount: number;
+  };
+};
+export class Resource implements PayloadResourceChild<PayloadResource> {
```

Comment:
Note:
Not decided right now if this stays a class or if I convert it to
a simple function which returns the JSON directly.